### PR TITLE
add zabbix

### PR DESCRIPTION
**Pull request by Mickael-Martin**

### CODE REVIEW

File: community.json [hunk 1]
```diff
@@ -1730,6 +1730,13 @@
         "revision": "795449cb43ccdba949c814746d6ea90eee0a7259",
         "state": "working",
         "url": "https://github.com/aymhce/yunohost_ynh"
+    },
+     "zabbix": {
+        "branch": "master",
+        "level": 2,
```
Comment (alexAubin) [inline community.json]: Please don't specify the level manually, it's meant to be the result of automatic test by the CI

File: community.json [hunk 2]
```diff
@@ -1730,6 +1730,13 @@
...
+        "revision": "9386c5b2",
```
Comment (alexAubin) [inline community.json]: I think you need to put the full commit id here, not just a short one ?

Comment (Mickael-Martin) [reply community.json]: Gitlab works with small ids like this : 
https://framagit.org/Mickael-Martin/zabbix_ynh/commits/master

File: community.json (hunk 1 above)
Comment (Mickael-Martin) [reply community.json]: Sorry, who make this test ? me ? you ? using https://github.com/YunoHost/package_check ?

Comment (frju365) [reply community.json]: Nobody. It's the CI who did automatic tests : https://ci-apps.yunohost.org/ci/

Comment (frju365) [reply community.json]: You have just to remove this line.

**Comment by alexAubin** [comment]
Thanks !

**Comment by Mickael-Martin** [comment]
Thank you too !

**Comment by Mickael-Martin** [comment]
sorry, why the app no appears on the community list ?

**Comment by Mickael-Martin** [comment]
in this list : https://app.yunohost.org/community.json

**Comment by alexAubin** [comment]
Thanks for reporting the issue.

I manually launched the build of the list (the community.json in this repo and the community.json you are pointing are two different files for various reasons)

I found this error : 
```
-> Error: unable to decompress object from https://framagit.org/Mickael-Martin/zabbix_ynh.git/objects/93/86c5b2 : Error -3 while decompressing data: incorrect header check
```

And it looks like this URL is indeed not working. Do you have an idea of the url we should use ?

**Comment by Mickael-Martin** [comment]
what is your command/code to obtain that ?
If you want try, I open a pushed mirror repo on https://github.com/Mickael-Martin/zabbix_ynh

I see the same issue with : 
  "wifiwithme": {
        "branch": "master",
        "level": 0,
        "revision": "14e01a3981bde7192e100b1f9cfa50e95d6cb89b",
        "state": "notworking",
        "url": "https://code.ffdn.org/ljf/wifiwithme_ynh"
},

**Comment by alexAubin** [comment]
To explain a bit what's going on, basically the "built" json file shall include the manifest of the app, among other things.

The relevant piece of code is here : https://github.com/YunoHost/apps/blob/master/list_builder.py#L252-L255

Sounds like there is indeed an issue with another app hosted on framagit (covoiturage_ynh) though I'm kinda sure it worked at some point ... Maybe the Gitlab API changed. Anyway, we shall investigate how to contact the API 'cause that's an important piece if we hope to move away from Github someday. The tricky part is that some other apps (e.g. wifiwithme) might be hosted on a Gogs with yet another API ...

**Comment by alexAubin** [comment]
I'm in the process of fixing the issue, but in the meantime, please consider changing the 'id' inside your `manifest.json` to just `zabbix` instead of `ynhzabbix` ... Otherwise people will see your app being called `ynhzabbix` which makes less sense than just `zabbix` :s

**Comment by Mickael-Martin** [comment]
Done ! Thanks !

**Comment by alexAubin** [comment]
Thanks !

Though please note that this won't be effective until the `revision` key is updated in `community.json` :sweat_smile: . (Yes, that's a lot of details to handle ..)

Nowadays we usually set the `revision` to `HEAD` to avoid updating it each time a new stable version is release. Though that means that you should probably have a `testing` branch for commits that are not suitable yet for production. See the note about HEAD here : https://github.com/YunoHost/Apps/#how-to-add-your-app-to-the-community-list

**Comment by frju365** [comment]
revision commit is : 9386c5b28e44ced928e886c00685d0f9a8486620 not the small one (we need the entire commit revision number - SHA number which is generated in fact)

**Comment by alexAubin** [comment]
(The short one works as well as the long one)

**Comment by frju365** [comment]
ok didn't know :P I prefer the complete one, as others PR has it.

**Pull request merged**